robottelo/ssh.py

```diff
@@ -27,6 +27,11 @@ def __init__(
             if output_format == 'json':
                 self.stdout = hammer.parse_json(stdout) if stdout else None
 
+    def __repr__(self):
+        tmpl = u'SSHCommandResult(stdout={stdout!r}, stderr={stderr!r}, ' + \
```

Comment:
Is better to avoid `\`, wrapping into `()` is better. and also the `+` is not needed.

```python
tmpl = (
    u'SSHCommandResult(stdout={stdout!r}, stderr={stderr!r}, '
    u'return_code={return_code!r}, output_format={output_format!r})'
)